content/resources/support/faq.md

```diff
@@ -76,12 +76,16 @@ after locations of our developer meetings.
 
 Yes you can... BUT depending on your Operating System, it is more or less difficult.
```

Comment:
Proposal:

"ECW is a proprietary file format. It is not included on all platforms out of the box."

.... platform specific notes ...